packaging/leapp.spec

```diff
@@ -25,7 +25,8 @@ Source0:    https://github.com/leapp-to/%{name}/archive/%{gittag}/%{name}-%{vers
 BuildArch:  noarch
 %if %{with python3}
 Requires: python3-%{name} = %{version}-%{release}
-%else
+%endif
+%if %{with python2}
```

Comment:
Hmm. Missing check for %fedora here. As this package will not be created on fedora now, it should be told, that this part is not relevant  for fedora `!0%{?fedora}` ... Another thing is that you are telling that the RPM can be ok with Py3 and Py2 but snactor can by build only for Py2 or Py3.

Another thing is, that you are telling that in case you build this for Python3 and Python2 - both has to be installed. Make the dependencies exclusive or let's discuss what are options. I am thinking here more about weak deps or rich dependencies to resolve the problem otherwise.

Reply:
This change indeed is a mistake. It should have been done for snactor.
/bin/leapp and /bin/snactor actually would be python3 because they are built last. 

I wanted to have both, actually since you need a way to run the tests for both. Python3 and Python2 and then you'd want to have both frameworks.

Reply:
I made the python2 / python3 part now a bit more obvious by creating snactor2 and snactor3 just in case.